src/Sextant/Navigation/ParameterViewStackService.cs

```diff
@@ -51,6 +52,7 @@ public ParameterViewStackService(IView view)
                 {
                     navigableViewModel
                         .WhenNavigatingTo(parameter)
+                        .ObserveOn(RxApp.MainThreadScheduler)
```

Comment:
May want to pass in a scheduler into the method (default to null), if null set to RxApp.MainThreadScheduler.

Reply:
Exposed `IView.MainThreadScheduler` and used the scheduler passed to the `IView` instance